src/utils/file.util.ts

```diff
@@ -0,0 +1,49 @@
+import { access, appendFile, constants, readFile } from "fs";
+import { injectable } from "inversify";
+
+const ENCODING = "utf8";
+
+@injectable()
+export class FileUtils {
+    public fileExistsAsync(filePath: string): Promise<boolean> {
+        return new Promise((resolve) => {
+            access(filePath, constants.F_OK, (error) => {
+                resolve(!error);
+            });
+        });
+    }
+
+    /**
+     * Appends a given string to a file for the given file path.
+     * This method creates the file if the file does not exist beforehand.
+     * @param filePath The path of the file that should be appended.
+     * @param appendance The string that should be appended to the file.
+     */
+    public appendStringInFileAsync(filePath: string, appendance: string): Promise<void> {
+        return new Promise((resolve, reject) => {
+            appendFile(filePath, appendance, (error) => {
```

Comment:
Change parameter name & update TSDoc accordingly

```suggestion
            appendFile(filePath, appendage, (error) => {
```